### ISSUE TITLE
Code Table Request - QUESTION - isotope value

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6330]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

I don't know what this means. Perhaps it's better as a 'detected' attribute?? Maybe it's fine?? ???

isotope value [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#isotope_value)

>The abundance of certain stable isotopes and chemical elements within organic and inorganic compounds.

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

```

 guid_prefix | count 
-------------+-------
 UAM:Mamm    |    30

```

 @amgunderson

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by ewommack** [comment]
with isobank now active this could be switch over to there? 
https://isobank.tacc.utexas.edu/

**Comment by jldunnum** [comment]
Agreed, developing reciprocal linkouts to Isobank (like we do to GenBank) has been on the to-develop list for a while I think. Joe Cook will be meeting with the Isobank group in the next couple weeks and could revive this discussion.

______________________________________________________________

Jonathan L. Dunnum Ph.D. (he, him, his)
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

Chair, Systematic Collections Committee, American Society of Mammalogists
Latin American Fellowship Committee, ASM

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131
________________________________

**Comment by campmlc** [comment]
We have isotope as a proposed examined_detected code table value. With an
IsoBank reciprocal link and the examined for: isotope attribute, that
should get folks to where they need? Or they could put the value in
detected: isotopes remarks.
The alternative is to put this as a part attribute - which is where it
really belongs.

**Comment by jldunnum** [comment]
[like]  Jonathan Dunnum reacted to your message:
________________________________

**Comment by dustymc** [comment]
> Is Arctos/collections really storing the isotopic values? 

Beats me, data below, didn't help me much...

If that's what this is, can it be moved to isobank? (I think these data pre-date the project.)

>suggesting we create parts 

That would only be appropriate if there's something to which a barcode might be stuck.

>example of an record 

https://arctos.database.museum/search.cfm?attribute_type_1=isotope%20value


>rope in

These all all @amgunderson data.

>reciprocal links

That's a question for isobank.

>link loans of a specific tissue type to a specific researcher

That's a matter of loan agreements. (Get good partIDs, add them to https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#part_identifier, require them to be used. https://github.com/ArctosDB/arctos/discussions/5231 might streamline something but it can't really change any of that.)

```

arctosprod@arctos>> select attribute_value,attribute_units  from attributes where attribute_type ='isotope value';
 attribute_value | attribute_units 
-----------------+-----------------
 -17.84          | 
 11.118          | 
 -18.41          | 
 -16.7           | 
 11.345          | 
 -16.28          | 
 11.87           | 
 11.627          | 
 -16.7           | 
 11.335          | 
 -17.41          | 
 -17.59          | 
 11.349          | 
 11.87           | 
 -16.43          | 
 -20             | 
 -15.96          | 
 11.944          | 
 11.768          | 
 -16.91          | 
 -16.41          | 
 12.289          | 
 -17.05          | 
 13.316          | 
 -16.83          | 
 11.692          | 
 12.026          | 
 -18.87          | 
 12.37           | 
 11.464          | 
(30 rows)

````

**Comment by Jegelewicz** [comment]
> That would only be appropriate if there's something to which a barcode might be stuck.

Or could have been stuck to - not all parts are around forever and some get lost...

**Comment by ewommack** [comment]
> Is Arctos/collections really storing the isotopic values?

I'd really suggest we try not to in the future. That metadata really influences how you can use the isotopic values you get from a sample. What machine you ran on, what your standard was, where the lab was situated, what prep they did of the sample, what conversion you used, etc. That will really affect your isotopic results, and the value can look the same but actually be telling you something really different with the metadata.
I liked the Isobank reporting set-up from a researcher side of things, because they have the metadata collection set-up. Then I could go to the lab that completed the analysis and say - hey I need these values too to report the data correctly.

**Comment by mkoo** [comment]
@ewommack yes that was a rhetorical question!-- agreed best to handle like a Genbank link and record

thx @campmlc for highlighting the issue here with the 30 UAM records with legacy data-- does Isobank have a mechanism for dealing with legacy data? (just wondering, like maybe they could have a record even with missing fields?) -- which may be what Beth was suggesting.... we would have to create 30 isobank records probably without the all the fields if they are not known, then link to UAM records

I still fear losing data however since the raw values would likely be relegated to remarks, so going to tentatively agree with @campmlc that we consider leaving this one. CT committee could change the description with best practise of using and linking to Isobank. Data could still be useful if only as guidance/ benchmark for future isotopic work.

**Comment by dustymc** [comment]
>losing data

See initial comment, my goal is to understand these data, I don't think we can tell if we're losing anything or not without that. I do not understand where the other comment came from, I think my goals were spelled out pretty clearly and they do not involve jumping to deleting!

>raw values would likely be relegated to remarks,

They already are! These data are not understandable without delving into remarks, and I'm not sure they are after either but hopefully that's just my ignorance showing.

>consider leaving this one

The big lump under the rug is a serious trip hazard; these things are piling up into cumulatively-blocking messes. If this is doing something that can be defined (or restructured to do something or whatever) and I'm just not getting it then clarifying the definition is a satisfactory solution. If this is the unstable mess it looks like from here, then let's PLEASE not just sweep it under the proverbial rug, at least not before exhausting all other options.

**Comment by Jegelewicz** [comment]
> fear losing data

There isn't much to lose? What do those numbers even mean? From digital data today:

"data linkages and relationships are indicators of digital provenance and quality"

Even with a date and determiner - those numbers are completely useless in Arctos. If there is no metadata in a usable format, the numbers are just numbers lacking any context and we are saving them for why?

Sorry for the rant, but we do need to start considering what data is important for managing a collection and what we should feel responsible for managing. We cannot be the repository for EVERYTHING. YES, we should pursue the path of getting these into isobank and setting up links to that data - that is the dream of the digitally extended specimen. Just tell me who I need to call...

**Comment by ewommack** [comment]
Christ at TACC was my main contact for getting the MVZ data into Isobank.

**Comment by campmlc** [comment]
They have divine guidance? 😏
